sass/_toolbar.scss

```diff
@@ -14,7 +14,7 @@
 
     .chatroom {
         .send-button {
-            background-color: var(--chatroom-head-bg-color);
+            background-color: var(--chatroom-send-button-bg);
```

Comment:
I see there is already a CSS propery `--muc-toolbar-btn-color`.

Looks like you can just use that, instead of adding a new one.